executor/runtime/types/pod.go

```diff
@@ -224,6 +225,42 @@ func (c *PodContainer) Capabilities() *corev1.Capabilities {
 	return c.capabilities
 }
 
+func (c *PodContainer) ContainerCapabilities(containerName string) ([]titusAPI.ContainerCapability, error) {
+	capabilities := []titusAPI.ContainerCapability{}
+
+	ann := podCommon.ContainerAnnotation(containerName, podCommon.AnnotationKeySuffixContainersCapabilities)
+	caps := []string{}
+	c.podLock.Lock()
+	val, ok := c.pod.Annotations[ann]
+	c.podLock.Unlock()
+	if ok && val != "" {
+		caps = strings.Split(val, ",")
+	}
+
+	hasFuse := false
+	for _, cStr := range caps {
+		c, ok := titusAPI.ContainerCapability_value["ContainerCapability"+cStr]
+		if !ok {
+			logrus.Errorf("unknown container capability %s", cStr)
+			return nil, fmt.Errorf("unknown container capability %s", cStr)
```

Comment:
Why not return a real error here?
We can be strict.

Reply:
Not sure what you mean here, can you elaborate?

Reply:
Oh sorry I misread